### PR TITLE
chore: node updates

### PR DESCRIPTION
**Pull request by paul121**
## Description

Closes: #451 

Opening this PR to see if the tests will pass when a PR is created from a branch in this repository, not a fork. *update*: yes, this was the issue, and tests pass.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] targeted `dev` branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] submit a PR against `master` branch after this one (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: .github/workflows/registryServer.yml [hunk 1]
```diff
@@ -72,13 +72,13 @@ jobs:
...
         env:
           MAILERLITE_API_KEY: foobar
           AIRTABLE_API_TOKEN: foobar
           DATABASE_URL: postgres://postgres:password@localhost:5432/server
           CSRF_SECRET: foobar
           CSRF_COOKIE_NAME: __Host-regen-dev.x-csrf-token
-        run: yarn build && yarn serve &
-      - name: 'run tests after waiting a bit'
-        run: yarn test
+        run: yarn serve & yarn test
```
Comment (paul121) [inline .github/workflows/registryServer.yml:84]: I'm just realizing that this change dumps all of the log messages from `yarn serve` into the same output as `yarn test`. It makes the test output a little bit harder to read.

But this change also makes it possible to debug any issues that happen with `yarn serve` if the application isn't starting. Previously there was no way to see this... so overall I think this is an improvement? What do you think @clevinson @blushi ?

Comment (blushi) [reply .github/workflows/registryServer.yml:84]: I have mixed up feelings, I think I would actually prefer to still see the tests output distinctly

Comment (paul121) [reply .github/workflows/registryServer.yml]: Cool, added a commit to keep the tests separate

**Comment by paul121** [comment]
Including the commit from #448

**Review by blushi — APPROVED** [review]
LGTM! thanks @wgwz and @paul121

**Pull request merged**